packages/sdk-core/src/modules/metrics/BacktraceMetrics.ts

```diff
@@ -84,12 +89,11 @@ export class BacktraceMetrics implements BacktraceModule {
     /**
      * Sends event to the metrics system.
      */
-    public send() {
+    public async send(abortSignal?: AbortSignal) {
         if (!this._sessionProvider.shouldSend()) {
             return false;
         }
-        this.sendUniqueEvent();
-        this._summedEventsSubmissionQueue.send();
+        await Promise.all([this.sendUniqueEvent(abortSignal), this._summedEventsSubmissionQueue.send(abortSignal)]);
```

Comment:
we do not need to wait on this tbh. We can just fire and forget that. Await can block API, so I think it would be better to not use it

Reply:
We cannot, because aborting will throw an unhandled promise rejection.